src/olympia/stats/management/commands/download_counts_from_file.py

```diff
@@ -149,44 +149,43 @@ def handle(self, *args, **options):
                 # Ignore completely invalid data.
                 continue
 
-            if id_or_slug.strip().isdigit():
+            if not is_valid_source(src, fulls=fulls, prefixes=prefixes):
+                continue
+
+            if id_or_slug.isdigit():
```

Comment:
Why did we get rid of `strip()`? Is it not needed? I guess there was no test coverage for it, or you would have had to update a test after removing it.

Reply:
we already `.strip()` it a few lines above.